runtime/tutor

```diff
@@ -1186,6 +1186,160 @@ multiple cursors, they won't be uncommented but commented again.
 
 
 
+=================================================================
+=                   12.1 USING MATCH MODE JUMP                  =
+=================================================================
+
+From normal mode, type "m" to enter the match mode. This is
+convenient to work with bracket pairs and the likes, and with
+their content.
```

Comment:
I am not a fan of the phrase _and the likes_, it pops up quite a bit but it doesn't really make sense to me, and maybe non native speakers will get confused by it.

Looking at the rest of the tutor it seems the done thing is to use the name of the key alone rather than in quotes, so `m` rather than `"m"`. I think it would be a good idea to change it everywhere in the PR.

```suggestion
To switch to match mode from normal mode, type m. This feature is particularly useful for handling bracket pairs and their contents.
```